src/versionUtils.ts

```diff
@@ -147,7 +147,7 @@ async function checkKamelCLIVersion(): Promise<string | void> {
 }
 
 export async function handleChangeRuntimeConfiguration() {
-	const runtimeSetting: string = stripLeadingV(config.getKamelRuntimeVersionConfig() as string);
+	const runtimeSetting: string = config.getKamelRuntimeVersionConfig() as string;
```

Comment:
isn"t the root issue caused by this cast to tring although it can be undefined?

Reply:
Actually it is. The call to getKamelRuntimeVersionConfig can return undefined so the cast to string is a bad thing.